components/automate-ui/src/app/modules/infra-proxy/create-environment-modal/create-environment-modal.component.html

```diff
@@ -58,7 +58,7 @@ <h2 slot="title">Create Environment</h2>
           </div>
           <div class="button-bar-box">
             <chef-button tertiary (click)="closeCreateModal()">Cancel</chef-button>
-            <chef-button [disabled]="!constraintArray.length || conflictError"  primary id="create-button-object-modal-1" (click)="createEnvironment()">
+            <chef-button [disabled]="!constraints.length || conflictError"  primary id="create-button-object-modal-1" (click)="createEnvironment()">
```

Comment:
Author note: Make names as long as necessary but no longer. We do not care it is an Array. It could be a Set or a List or a Widget; we do care that it is a bunch of constraints.